buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -730,6 +730,9 @@ class BeamModulePlugin implements Plugin<Project> {
       project.apply plugin: 'propdeps-maven'
       project.apply plugin: 'propdeps-idea'
 
+      // Defines Targets for sonarqube analysis reporting.
+      project.apply plugin: "org.sonarqube"
```

Comment:
Do we really want sonarqube to be applied on root and all subproject? Usually it is sufficient to add only to the root project?

If so, I would expect the sonarqube plugin to be part of plugins class path, see
https://github.com/apache/beam/blob/master/buildSrc/build.gradle#L39-L53

something like
`runtime 'org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:2.7'`

Reply:
Revisiting, we should simply drop that?

Reply:
I tried doing this, but never managed to import plugin properly. I always hit the bar where it is either imported multiple times (my assumption), or can't be imported at all.

Reply:
Hmmm.... unfortunately I do not understand the issue.

The plugin is applied for root project in build.gradle. Sonarqubes docs states that this is the way to handle multi project builds [1].

But as you tested that this it what gets us working, we should just proceed with merging.

Thx for your effort here!

[1] https://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner+for+Gradle#AnalyzingwithSonarQubeScannerforGradle-AnalyzingMulti-ProjectBuilds